example/lib/screens/user_screen.dart

```diff
@@ -406,6 +399,19 @@ class Home extends StatelessWidget {
         fontSize: 16.0);
   }
 
+  authenticatedUserProfile(BuildContext context) async {
```

Comment:
This method shows toast with profile id, name of the method should be different, like `showAuthenticatedUserProfile` or similar.